### PR TITLE
Add venue information and hotel details

### PR DESCRIPTION
**Pull request by roadsideseb**
I've updated the venue on the home page and added the "venue" page with a few hotel options and some basic details on accommodation.

Feel free to update or suggest changes. I'm not super familiar with the area and the hotel options might not reflect the variety that we should be pointing towards for our attendees.

### CODE REVIEW

File: templates/venue.jade [hunk 1]
```diff
@@ -81,7 +76,7 @@ append bottom
     )
 
     script.
-        var kaneHallCoordinates = [47.6562467,-122.3093368];
+        var kaneHallCoordinates = [45.5191784,-122.654018];
```
Comment (EstiShay) [inline templates/venue.jade]: Do we need to keep this variable named after Kane Hall?

File: templates/venue.jade [hunk 2]
```diff
@@ -99,9 +94,9 @@ append bottom
         }).addTo(map);
 
         var kaneHall = L.marker(kaneHallCoordinates).bindPopup(
```
Comment (EstiShay) [inline templates/venue.jade]: Should we rename this `revHall` instead of `kaneHall`?

**Review by EstiShay — COMMENTED** [review]
Can we post this without hotel recommendations?

File: templates/venue.jade (hunk 1 above)
Comment (roadsideseb) [reply templates/venue.jade]: Good catch. I fixed it.

**Pull request merged**